HW_module_9.py

```diff
@@ -0,0 +1,158 @@
+import os
+import xml.etree.ElementTree as et
+from HW_module_6 import News, Advertising, Guess, FromAnotherSource
+from HW_module_7 import WordsCount, LettersCount
+from HW_module_8 import FileInsert, FromJson
+
```

Comment:
The recommendations are the same as for JSON. Let's have a separate call to review the decomposition strategy together.